plugins/dell-kestrel/fu-dell-kestrel-hid-device.c

```diff
@@ -249,19 +249,29 @@ fu_dell_kestrel_hid_device_write_firmware_pages(FuDellKestrelHidDevice *self,
 						      page_ack_time,
 						      &error_local)) {
 			/* A buggy device may fail to send an acknowledgment receipt
-			   after the last page write, resulting in a timeout error.
-
-			   This is a known issue so waive it for now.
+			   after the last page write, resulting in a timeout error or a pipe
+			   error on reading. This is a known issue so waive it for now.
 			*/
 			if (dev_type == FU_DELL_KESTREL_EC_DEV_TYPE_LAN &&
-			    j == fu_chunk_array_length(pages) - 1 &&
-			    g_error_matches(error_local,
-					    G_USB_DEVICE_ERROR,
-					    G_USB_DEVICE_ERROR_TIMED_OUT)) {
-				g_debug("ignored error: %s", error_local->message);
-				fu_progress_step_done(progress);
-				continue;
+			    j == fu_chunk_array_length(pages) - 1) {
+				if (g_error_matches(error_local,
+						    G_USB_DEVICE_ERROR,
+						    G_USB_DEVICE_ERROR_NOT_SUPPORTED) ||
+#ifdef G_USB_DEVICE_ERROR_BUSY
```

Comment:
it's not a define, it's an enum member. You need `#if G_USB_CHECK_VERSION(0,4,8)` instead